core/src/main/java/com/vityuk/ginger/loader/PropertiesLocalizationLoader.java

```diff
@@ -45,11 +46,9 @@
     private static final CharMatcher COMMENT_MATCHER = CharMatcher.anyOf("#!");
     private static final CharMatcher LINE_SEPARATOR_MATCHER = CharMatcher.anyOf("\n\r");
     private static final CharMatcher NOT_LINE_SEPARATOR_MATCHER = LINE_SEPARATOR_MATCHER.negate();
-    private static final CharMatcher WHITESPACE_MATCHER =
-            CharMatcher.BREAKING_WHITESPACE.and(NOT_LINE_SEPARATOR_MATCHER);
+    private static final CharMatcher WHITESPACE_MATCHER = CharMatcher.BREAKING_WHITESPACE.and(NOT_LINE_SEPARATOR_MATCHER);
```

Comment:
What is your line width? I thing mine is limited to 120 characters which fits well into 15" screen with good resolution when viewed from github code browser.